tests/integration/session_management/test_reservation.py

```diff
@@ -165,6 +165,95 @@ def test___sessions_reserved___get_connections_by_instrument_type___connections_
         ]
 
 
+def test___sessions_reserved_by_pin_group___get_connections_by_pins___connections_returned(
```

Comment:
https://github.com/ni/measurementlink-python/pull/639#discussion_r1514475411

Should the `Given` clause be "sessions_reserved_with_pin_groups" or "sessions_reserved_using_pin_groups"?

Reply:
I have updated the _given_ clause to `sessions_reserved_*using*_pin_groups`. Additionally, I have modified the _return_ clause to `returns_connection(s)`, as discussed in a different thread related to another PR.